src/physics/fermions.jl

```diff
@@ -34,7 +34,7 @@ end
 
 isfermionic(i::Index) = has_fermionic_subspaces(i)
 
-has_fermionic_subspaces(is::IndexSet) = false
+has_fermionic_subspaces(is::Union{IndexSet,NTuple{N,Index}}) where {N} = false
 
 function has_fermionic_subspaces(is::Union{QNIndexSet,NTuple{N,QNIndex}}) where {N}
```

Comment:
You could use `QNIndices` here:
https://github.com/ITensor/ITensors.jl/blob/23bc031e45d1fac0bbc839e524f85c740ac48b66/src/qn/qnindexset.jl#L4

Reply:
Great